app/models/review.rb

```diff
@@ -1,4 +1,8 @@
 class Review < ApplicationRecord
   belongs_to :user
   belongs_to :instrument
+
+  validates :rate, presence: true
+  validates :rate, numericality: { only_integer: true }
```

Comment:
Maybe should we allow the rating from 1 to 5 ?

  validates :rate, numericality: { only_integer: true,  greater_than_or_equal_to: 1, less_than_or_equal_to: 5}

Reply:
yesssss. nice one. will add